src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faShoppingCart,
+  faInfo,
+  faSearch,
+} from '@fortawesome/free-solid-svg-icons'
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <NavLink aria-label="Go to the start page to search for shops." to="/">
+        <Icon className="icon" icon={faSearch} />
+      </NavLink>
+      <NavLink
+        aria-label="See the details of a supermarket that you have chosen beforehand."
```

Comment:
Nice that you put in the aria-labels!

Reply:
Yes, would like to learn more about access. but one step at a time :)